cluster-operator/src/main/java/io/strimzi/operator/cluster/model/AbstractModel.java

```diff
@@ -156,6 +156,19 @@
     private static final String ENV_VAR_HTTP_PROXY = "HTTP_PROXY";
     private static final String ENV_VAR_HTTPS_PROXY = "HTTPS_PROXY";
     private static final String ENV_VAR_NO_PROXY = "NO_PROXY";
+
+    protected static final String CO_ENV_VAR_CUSTOM_BRIDGE_DEPLOYMENT_LABELS = "STRIMZI_CUSTOM_KAFKA_BRIDGE_LABELS";
+    protected static final String CO_ENV_VAR_CUSTOM_KAFKA_DEPLOYMENT_LABELS = "STRIMZI_CUSTOM_KAFKA_LABELS";
+    protected static final String CO_ENV_VAR_CUSTOM_KAFKA_EXPORTER_DEPLOYMENT_LABELS = "STRIMZI_CUSTOM_KAFKA_EXPORTER_LABELS";
+    protected static final String CO_ENV_VAR_CUSTOM_ZOOKEEPER_DEPLOYMENT_LABELS = "STRIMZI_CUSTOM_ZOOKEEPER_LABELS";
+    protected static final String CO_ENV_VAR_CUSTOM_MIRROR_MAKER_DEPLOYMENT_LABELS = "STRIMZI_CUSTOM_KAFKA_MIRROR_MAKER_LABELS";
+    protected static final String CO_ENV_VAR_CUSTOM_MIRROR_MAKER2_DEPLOYMENT_LABELS = "STRIMZI_CUSTOM_KAFKA_MIRROR_MAKER2_LABELS";
+    protected static final String CO_ENV_VAR_CUSTOM_JMX_TRANS_DEPLOYMENT_LABELS = "STRIMZI_CUSTOM_JMX_TRANS_LABELS";
+    protected static final String CO_ENV_VAR_CUSTOM_CONNECT_DEPLOYMENT_LABELS = "STRIMZI_CUSTOM_KAFKA_CONNECT_LABELS";
+    protected static final String CO_ENV_VAR_CUSTOM_CONNECT_BUILD_DEPLOYMENT_LABELS = "STRIMZI_CUSTOM_KAFKA_CONNECT_BUILD_LABELS";
+    protected static final String CO_ENV_VAR_CUSTOM_ENTITY_OPERATOR_DEPLOYMENT_LABELS = "STRIMZI_CUSTOM_ENTITY_OPERATOR_LABELS";
+    protected static final String CO_ENV_VAR_CUSTOM_CRUISE_CONTROL_DEPLOYMENT_LABELS = "STRIMZI_CUSTOM_CRUISE_CONTROL_LABELS";
```

Comment:
These are all supposed to be Pod labels. So calling the variables `DEPLOYMENT_LABELS` is IMHO confusing.